homeassistant/components/huawei_lte/device_tracker.py

```diff
@@ -185,7 +185,8 @@ class HuaweiLteScannerEntity(HuaweiLteBaseEntity, ScannerEntity):
     _extra_state_attributes: dict[str, Any] = field(default_factory=dict, init=False)
 
     @property
-    def _entity_name(self) -> str:
+    def name(self) -> str:
```

Comment:
Why use this instead of something like the sensor below where `__post_init__` which sets `_attr_name` to the same value?  Is it because the hostname could change and we want the entity to update?

Reply:
It could change, and in any case the hostname will only be available after the first successful data update.